editor/src/components/inspector/inspector.tsx

```diff
@@ -231,7 +234,10 @@ export function shouldInspectorUpdate(
   strategyState: StrategyState,
   elementsToRerender: ElementsToRerender,
 ): boolean {
-  return !isStrategyActive(strategyState) && elementsToRerender === 'rerender-all-elements'
+  return (
+    (!isStrategyActive(strategyState) && elementsToRerender === 'rerender-all-elements') ||
+    isKeyboardAbsoluteStrategy(strategyState.currentStrategy)
```

Comment:
I suspect we should be checking for any keyboard strategy here, rather than just absolute move / resize 🤔

Reply:
I skipped keyboard reorder strategy now because it doesn't change anything in the inspector sections